Description: Fix type issues on 64bit platforms
 Causes runtime failure of gptsync, for obvious reasons.
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/gptsync/gptsync.h
===================================================================
--- refit-0.14.orig/gptsync/gptsync.h	2010-04-19 18:29:15.591930983 +0200
+++ refit-0.14/gptsync/gptsync.h	2010-04-19 18:29:50.255932164 +0200
@@ -77,7 +77,7 @@
 typedef unsigned int        UINTN;
 typedef unsigned char       UINT8;
 typedef unsigned short      UINT16;
-typedef unsigned long       UINT32;
+typedef unsigned int        UINT32;
 typedef unsigned long long  UINT64;
 typedef void                VOID;
 
Index: refit-0.14/gptsync/lib.c
===================================================================
--- refit-0.14.orig/gptsync/lib.c	2010-04-19 18:29:15.606932040 +0200
+++ refit-0.14/gptsync/lib.c	2010-04-19 18:29:50.255932164 +0200
@@ -268,7 +268,7 @@
         Print(L" No GPT partition table present!\n");
         return 0;
     }
-    if (header->spec_revision != 0x00010000UL) {
+    if (header->spec_revision != 0x00010000) {
         Print(L" Warning: Unknown GPT spec revision 0x%08x\n", header->spec_revision);
     }
     if ((512 % header->entry_size) > 0 || header->entry_size > 512) {
